pkg/prometheus/testdata/AdditionalAlertRelabelConfigs.golden

```diff
@@ -1,3 +1,7 @@
 - action: drop
   source_labels: [__meta_kubernetes_node_name]
-  regex: spot-(.+)
\ No newline at end of file
+  regex: spot-(.+)
+- sourceLabels: [label-a, label-b]
+  separator: ""
+  action: replace
+  targetLabel: custom-label
```

Comment:
```suggestion
- source_labels: [label-a, label-b]
  separator: ""
  action: replace
  target_label: custom-label
```

Reply:
I thought the input file followed the operator's format. And the output (expected) file followed Prometheus's format. But from your suggestion, it means both follow Prometheus's format?

Reply:
correct, it's because the test is about additionalScrapeConfigs which hold native Prometheus configs.

Reply:
Hmm, then do you think this test can verify our fix? I thought we needed a test regarding the operator's format? 🤔 Like, before the operator's format doesn't support `""` but now it does?

Reply:
The operator appends the additional config you specify by name (I think?) in the final generated config. In this, we process the input config (user defined). I think what you're trying to test is that the generated config which contains the appended additional config still contains the empty field instead of omitting it entirely.